valhalla/worker.h

```diff
@@ -16,19 +16,19 @@ namespace valhalla {
 
 // to use protobuflite we cant use descriptors which means we cant translate enums to strings
 // and so we reimplement the ones we use here
-bool Options_Action_Parse(const std::string& action, Options::Action* a);
```

Comment:
this is the actual change. we rename these functions to avoid name collisions with newer versions of protobuf that actually even in the lite generated code will provide these